test/e2e_node/e2e_remote.go

```diff
@@ -82,12 +82,14 @@ func GetHostnameOrIp(hostname string) string {
 // the binaries k8s required for node e2e tests
 func CreateTestArchive() (string, error) {
```

Comment:
This function is called in [getArchive](https://github.com/Random-Liu/kubernetes/blob/1c5bd5540e0dd3aa9f8215a943cf66f01e8da39a/test/e2e_node/runner/run_e2e.go#L254), which is called [here](https://github.com/Random-Liu/kubernetes/blob/1c5bd5540e0dd3aa9f8215a943cf66f01e8da39a/test/e2e_node/runner/run_e2e.go#L207) and the caller doesn't check the error at all. Would this cause any problem?